### PR TITLE
DOP-3135 - add project version to page title, apply defaults

### PR DESCRIPTION
**Pull request by seungpark**
### Stories/Links:

DOP-3135

### Current Behavior:

[cloud docs](https://www.mongodb.com/docs/atlas/)
[server docs without version in prod](https://www.mongodb.com/docs/manual/)
[docs landing page with no RST title has double value in <title>](https://www.mongodb.com/docs/search/)

### Staging Links:

[cloud docs should have no change in page title since there are no versions](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/master/cloud-docs/seung.park/DOP-3135/index.html)
[server docs now have version added](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/v7.0/docs/seung.park/DOP-3135/index.html)
[docs landing with fixed fallback default](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/master/landing/seung.park/DOP-3135/search/index.html)

### Notes:
- To verify, inspect the page's Elements and search for `<title>` element

### README updates

- - [ ] This PR introduces changes that should be reflected in the README, and I have made those updates.
- - [X] This PR does not introduce changes that should be reflected in the README

### CODE REVIEW

File: tests/unit/Head.test.js [hunk 1]
```diff
@@ -148,4 +149,52 @@ describe('Head', () => {
...
+      expect(title.innerHTML).toBe(`${metadata.title} ${metadata.branch}`);
+    });
+
+    it('only applies branch versions starting with a version number (v)', function () {
+      metadata.branch = 'master';
+      useSnootyMetadata.mockImplementation(() => ({ ...metadata }));
+      const { container } = render(<Head pageContext={pageContext} />);
+      const title = container.querySelector('title');
+      expect(title.innerHTML).toBe(`Get Started with  - ${metadata.title}`);
+      // metadata.version = 'master'
```
Comment (rayangler) [inline tests/unit/Head.test.js:186]: Do we still need this comment?

File: src/utils/get-site-title.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { getNestedValue } from './get-nested-value';
+
+/**
+ * Given property metadata,
+ * return the page title as a string
```
Comment (rayangler) [inline src/utils/get-site-title.js:5]: Return the page title or the site title?

**Review by rayangler — APPROVED** [review]
I think this generally lgtm. Just wanted to flag a couple potential things for cleanup

File: src/utils/get-site-title.js (hunk 2 above)
Comment (seungpark) [reply src/utils/get-site-title.js:5]: good catch!

File: tests/unit/Head.test.js (hunk 1 above)
Comment (seungpark) [reply tests/unit/Head.test.js:186]: ha caught my misnaming of branch 😛

**Pull request merged**